src/pages/directors.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import { List } from '@material-ui/core'
+import { useStaticQuery, graphql, Link } from 'gatsby'
+import Layout from '../components/layout'
+import DirectorListItem from '../components/DirectorListItem'
+import SEO from '../components/seo'
+import SearchBar from '../components/SearchBar'
+
+const DirectorsPage = ({ ...props }) => {
+  const data = useStaticQuery(graphql`
+    {
+      allDirectorsJson {
+        nodes {
+          directors {
+            id
+            first_name
+            last_name
+          }
+        }
+      }
+    }
+  `)
+  const [directorsArr] = data.allDirectorsJson.nodes.map(node => node.directors)
+
+  const [searchResults, setSearchResults] = React.useState(directorsArr)
+
+  const filterDirectors = value => {
```

Comment:
Maybe it's better to take out comparator functionality from component - it won't be recreated in every render and the component would be more concise

Reply:
Yes, you are right! Done